src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -189,6 +206,20 @@ $(function () {
                  });
   }
 
+  function escapeHtml(string) {
```

Comment:
It looks good, but I don't want to add another library for just one function. Maybe, we can add later if we need more needs.